sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -713,6 +735,11 @@ public partial class BlobLeaseRequestConditions : Azure.RequestConditions
         public BlobLeaseRequestConditions() { }
         public string TagConditions { get { throw null; } set { } }
     }
+    public partial class BlobLegalHoldInfo
```

Comment:
This class is returned by `SetLegalHold()`, and exists for future-proofing.  Switch from returning a `bool` to returning a `BlobLegalHoldInfo` in the future would be a breaking change.

Reply:
Shouldn't this be `BlobLegalHoldResult` ?

Reply:
`*Result*` is compliant with https://azure.github.io/azure-sdk/general_design.html#model-types

Reply:
Renamed to `*Result`